### PR TITLE
Use export_local_property_definitions closure compiler option

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: ProjectSample/projectsample/static/js/maincontroller.js [hunk 1]
```diff
@@ -31,10 +31,16 @@ app.MainController = function(gettextCatalog, langUrlTemplate) {
   this.langUrlTemplate_ = langUrlTemplate;
 
   /**
-   * @private
+   * @type {string|undefined}
+   * @export
+   */
+  this.lang = undefined;
```
Comment (elemoine) [inline ProjectSample/projectsample/static/js/maincontroller.js]: We could use the empty string as the initial value, and just `string` for the type.

**Comment by elemoine** [comment]
Just a minor comment. Otherwise looks good to me.

**Comment by fredj** [comment]
thanks for the review

File: ProjectSample/projectsample/static/js/maincontroller.js [hunk 2]
```diff
@@ -31,10 +31,16 @@ app.MainController = function(gettextCatalog, langUrlTemplate) {
   this.langUrlTemplate_ = langUrlTemplate;
 
   /**
-   * @private
+   * @type {string}
+   * @export
+   */
+  this.lang;
```
Comment (elemoine) [inline ProjectSample/projectsample/static/js/maincontroller.js]: Any reason for not initializing it to the empty string?

**Pull request merged**